src/components/views/settings/KeyBackupPanel.js

```diff
@@ -146,13 +150,35 @@ export default class KeyBackupPanel extends React.PureComponent {
             return <Spinner />;
         } else if (this.state.backupInfo) {
             let clientBackupStatus;
+            let buttons;
+
             if (MatrixClientPeg.get().getKeyBackupEnabled()) {
-                clientBackupStatus = _t("This device is using key backup");
+                clientBackupStatus = <div>
+                    <p>{encryptedMessageAreEncrypted}</p>
+                    <p>{_t("This device is backing up your keys. ✅")}</p>
```

Comment:
somewhat of an academic question: should the emoji be translatable? are we aware of any languages where the placement of said emoji will affect the sentence structure/message? 

also should we be using emojione for consistency?

Reply:
Very good point - fixed